apps/ex_nvr_web/lib/ex_nvr_web/live/recordings_list_live.ex

```diff
@@ -136,6 +137,8 @@ defmodule ExNVRWeb.RecordingListLive do
 
   @impl true
   def mount(params, _session, socket) do
+    PubSub.subscribe(ExNVR.PubSub, "recordings")
```

Comment:
It's better to hide the name of the topic. A function in `recordings.ex` would do the job
```elixir
def subscribe_to_recording_events()
  PubSub.subscribe(ExNVR.PubSub, @recordings_topic)
end